05-JWT-Basics/starter/errors/unauthenticated.js

```diff
@@ -0,0 +1,11 @@
+const CustomAPIError = require("./custom-error");
+const { StatusCodes } = require("http-status-codes");
+
+class UnauthenticatedError extends CustomAPIError {
+  constructor(message) {
+    super(message);
+    this.statusCode = StatusCodes.UNAUTHERIZED;
```

Comment:
`UNAUTHORIZED`! Watch the spelling. 

Extra challenge: can we create a wrapper around the StatusCodes library so that it throws an exception for an invalid property access like this instead of just moving on with an `undefined` floating around? We can discuss this or slack me because this one is a tricky challenge!

Reply:
Taking a look at the function, this also causes a bug where a server error will occur instead of sending a 400 error, since StatusCodes.UNAUTHERIZED is undefined. The behavior should be fixed if you just fix the spelling.